packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md

```diff
@@ -20,13 +20,26 @@ You can customize your Okta org URL by replacing the Okta domain name with your
 
 Each organization also has an administrator URL to sign in to the administrator console. The admin URL is the subdomain plus `-admin` (for example, `companyname-admin.okta.com`). If you have [customized your org URL](/docs/guides/custom-url-domain/overview/), access the admin console using your un-customized URL.
 
+### Preview and Production
+Okta orgs come in two varieties: preview orgs and production orgs.
+
+Preview orgs allow you to see the next release early and play with Beta features. Preview orgs include [Beta](/docs/reference/releases-at-okta/#beta) and [Early Access](/docs/reference/releases-at-okta/#early-access-ea) (EA) features by invitation and include all features that are [Generally Available](/docs/reference/releases-at-okta/#general-availability-ga) (GA).
+
+> **Note:** You can't convert Preview orgs into Production orgs, and you can't convert Production orgs into Preview orgs.
```

Comment:
I would phrase this as "Preview orgs cannot be converted..." just to make it clear that this isn't a self-service limitation.

i.e. it's not that _you_ can't convert one into another, it's that _no one_ can.

Reply:
Done.